### PR TITLE
Helpful info for PRs: build directive.

### PR DESCRIPTION
**Pull request by anastasiamac**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -350,6 +350,14 @@ Be sure to have a look at:
 
 https://help.github.com/articles/using-pull-requests
 
+Sanity checking PRs and unit tests
+----------------------
+
+All PRs run pre-merge check - unit tests and small but representative sample of functional tests.
```
Comment (Veebers) [inline CONTRIBUTING.md]: . . . and a small but . . .

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -350,6 +350,14 @@ Be sure to have a look at:
...
+This check is re-run anytime the PR changes, for example a new commit is added.
```
Comment (Veebers) [inline CONTRIBUTING.md]: for example when a ...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -350,6 +350,14 @@ Be sure to have a look at:
...
+
+You can also initiate this check by using !!build!! comment on PR.
```
Comment (Veebers) [inline CONTRIBUTING.md]: by commenting !!build!! on the PR.

**Comment by anastasiamac** [comment]
unrelated failure: github.com/juju/juju/api/watcher.TestAll

**Comment by anastasiamac** [comment]
$$merge$$

**Pull request merged**